loopy/target/pyopencl_execution.py

```diff
@@ -274,6 +274,9 @@ def get_invoker_uncached(self, kernel, codegen_result):
         generator = PyOpenCLExecutionWrapperGenerator()
         return generator(kernel, codegen_result)
 
+    def get_wrapper_generator(self):
+        return PyOpenCLExecutionWrapperGenerator()
```

Comment:
A concern I have is that I'm not willing to call any of this a public API surface, which means the interface is allowed to change at any time. That leaves upstream consumers of this interface in an uncomfortable spot. I'd somehow prefer if we could come up with an interface that we'd be willing to support long term. This is not that interface, not in the least because the existence (let alone the mode of generation) of the executor is an implementation detail.

If you're OK with this, I'm OK with merging this PR (but I will break this interface if I need to).

Reply:
I'd be fine with that.